src/opera/pge/dist_s1/dist_s1_pge.py

```diff
@@ -31,12 +33,226 @@ class DistS1PreProcessorMixin(PreProcessorMixin):
 
     This particular pre-processor is currently a stub implementation, inheriting from the base pre-processor mixin
     and adding nothing at this time. New functionalities will be added as new versions of the DIST-S1 SAS are released.
-
     """
 
     _pre_mixin_name = "DistS1PreProcessorMixin"
     _valid_input_extensions = (".tif",)
 
+    _rtc_pattern = re.compile(r'(?P<id>(?P<project>OPERA)_(?P<level>L2)_(?P<product_type>RTC)-(?P<source>S1)_'
+                              r'(?P<burst_id>\w{4}-\w{6}-\w{3})_(?P<acquisition_ts>\d{4}\d{2}\d{2}T\d{2}\d{2}\d{2}Z)'
+                              r'_(?P<creation_ts>\d{4}\d{2}\d{2}T\d{2}\d{2}\d{2}Z)_(?P<sensor>S1A|S1B|S1C)_'
```

Comment:
One last thing we can do here now is compress the `\d{4}\d{2}\d{2}T\d{2}\d{2}\d{2}` to just `\d{8}T\d{6}`